example/material.json

```diff
@@ -66,6 +66,46 @@
             "value": 0.71
         }
     ],
+    "molecularProperties": {
+        "charge": 0,
+        "multiplicity": 1,
+        "covalentBonds": [
+            {
```

Comment:
We can use `include` statements like in other examples to avoid duplicating the data